src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationPatch.kt

```diff
@@ -68,6 +56,47 @@ class SpoofSignatureVerificationPatch : BytecodePatch(
             }
         } ?: return ProtobufParameterBuilderFingerprint.toErrorResult()
 
+
+        // Force the video time and chapters to show up when using the seekbar.
```

Comment:
Document the strategy of this patch, its purpose, why it is necessary, and how it works. Someone passing by this will question why this patch exists and how it works. Take an example from [here](https://github.com/revanced/revanced-patches/blob/main/src/main/kotlin/app/revanced/patches/youtube/misc/litho/filter/patch/LithoFilterPatch.kt#L38-L72).